pandas/core/generic.py

```diff
@@ -3074,11 +3074,12 @@ def to_latex(
             centered labels (instead of top-aligned) across the contained
             rows, separating groups via clines. The default will be read
             from the pandas config module.
-        caption : str, optional
-            The LaTeX caption to be placed inside ``\caption{}`` in the output.
+        caption : str, tuple, optional
```

Comment:
str or tuple, optional (change the signature as well)
Optional[Union[str, Tuple[str, str]]]

Reply:
Done. I updated typing on caption in DataFrameFormatter.to_latex() and LatexFormatter.